src/test/java/io/jenkins/plugins/analysis/warnings/ParsersITest.java

```diff
@@ -737,7 +737,17 @@ public void shouldFindAllJavaIssues() {
         shouldFindIssuesOfTool(2 + 1 + 1 + 2, new Java(), "javac.txt", "gradle.java.log", "ant-javac.txt", "hpi.txt");
     }
 
-    /** Runs the CssLint parser on an output file that contains 51 issues. */
+    /**
+     * Runs the Kotlin parser on several output files that contain 4 issues.
```

Comment:
contains 1 issue

Reply:
Where do I upload the logo and do we want only a logo or a log and text with it ?

Reply:
Where do I upload the logo and do we want only a logo or a log and text with it ?

Reply:
The logos are in https://github.com/jenkinsci/warnings-ng-plugin/tree/master/src/main/webapp/icons. Please add a line that references the license somehow in the License.txt file. 

You can have a look at the https://github.com/jenkinsci/warnings-ng-plugin/blob/master/src/main/java/io/jenkins/plugins/analysis/warnings/AndroidLint.java to see how icons are used.

If everything is working correctly you will see the result in https://github.com/jenkinsci/warnings-ng-plugin/blob/master/SUPPORTED-FORMATS.md (after running the `ToolLister`).